src/Operation/CreateIndexes.php

```diff
@@ -202,6 +213,16 @@ private function executeCommand(Server $server)
             'indexes' => $this->indexes,
         ];
 
+        if (isset($this->options['commitQuorum'])) {
+            /* Drivers MUST manually raise an error if this option is specified
+             * when creating an index on a pre 4.4 server. */
+            if (! server_supports_feature($server, self::$wireVersionForCommitQuorum)) {
+                throw new CommandException('Invalid field specified for createIndexes command: commitQuorum', 2);
```

Comment:
Where is code `2` come from?

I think this should throw an UnsupportedException via a new factory method for consistency.